### ISSUE TITLE
Packaging

### ISSUE THREAD

**Issue by Bzero** [Bzero/typstwriter#25]
A wider variety of distribution packages should be available, e.g:

- [ ] Appimage
- [ ] Windows Installer

**Comment by OR-gatti** [comment]
Are you planning to make the app for Mac?

**Comment by Bzero** [comment]
Yes, Mac support would definitely be great. In principle most of the application should probably already work on Mac, unfortunately I can not test it myself. 

It you want to give it a go I would be glad to get feedback.
